README.adoc

```diff
@@ -136,12 +136,14 @@ $ odo push
 $ odo url create
 $ curl <generated URL>
 ----
-. When finished, remove your component from `Minishift` and stop your `Minishift` cluster.
+. When finished, remove your component from `Minishift` or `CodeReady Containers`, and stop your cluster if you are finished with it.
 +
 [source,bash]
 ----
 $ odo delete <component>
 $ minishift stop
+# or
+$ crc stop
```

Comment:
Can it be more clean like
```
+
[source,bash]
----
$ odo delete <component>
$ minishift stop
----
+
or
+
[source,bash]
----
$ odo delete <component>
$ crc stop
----
```
<img width="989" alt="Screen Shot 2019-10-03 at 1 09 28 PM" src="https://user-images.githubusercontent.com/19851115/66107941-0e2feb80-e5df-11e9-8fe1-0ebd9353f03b.png">